python/lsst/ip/isr/overscan.py

```diff
@@ -617,7 +617,8 @@ def maskOutliers(self, imageArray):
         maskedArray : `numpy.ma.masked_array`
             Masked image marking outliers.
         """
-        lq, median, uq = np.percentile(imageArray, [25.0, 50.0, 75.0], axis=1)
+        lq, median, uq = np.percentile(np.ma.getdata(imageArray),
+                                       [25.0, 50.0, 75.0], axis=1)
```

Comment:
I was worried that this wouldn't work if it wasn't a masked array, but it does, so great!